### PR TITLE
Add languages to ll

### PR DESCRIPTION
**Pull request by SusannaShu**
added french language learning

### CODE REVIEW

File: server/server_config.py [hunk 1]
```diff
@@ -40,7 +40,7 @@
 # Prod:
 database_uri = "mongodb://localhost:27017"
 server_port = 8080
-path_modifier = ""
+path_modifier = "/dev4"
```
Comment (nic-olo) [inline server/server_config.py:43]: Same as above.

**Comment by CaydenPierce** [comment]
Is this PR going to add the other languages as well?

Let's name PRs better to explain what they're doing.

**Comment by CaydenPierce** [comment]
@SusannaShu 

Did you test all the languages to make sure the dictionaries work on the backend?

The build.gradle change is still in here, please remove it.

Otherwise looks good!

**Comment by CaydenPierce** [comment]
Also, there needs to an SGM commit that adds the new languages to the ASR files.

**Pull request closed without merging**